src/citrine/informatics/generative_design.py

```diff
@@ -71,22 +106,31 @@ class GenerativeDesignInput(Serializable['GenerativeDesignInput']):
         The number of initial mutations that will be attempted per seed.
         IMPORTANT, the total number of molecules generated will likely be lower than this value.
         Some mutations may be duplicates or may not meet the fingerprint similarity threshold.
+    structure_exclusions: List[StructureExclusion]
+        The structure exclusions used to limit molecule mutations.
+        If None, no structure exclusions will be used.
 
     """
 
     seeds = properties.List(properties.String(), 'seeds')
     fingerprint_type = properties.Enumeration(FingerprintType, "fingerprint_type")
     min_fingerprint_similarity = properties.Float("min_fingerprint_similarity")
     mutation_per_seed = properties.Integer("mutation_per_seed")
+    structure_exclusions = properties.List(
+        properties.Enumeration(StructureExclusion),
+        "structure_exclusions"
+    )
 
     def __init__(
         self, *,
         seeds: List[str],
         fingerprint_type: FingerprintType,
         min_fingerprint_similarity: float,
-        mutation_per_seed: int
+        mutation_per_seed: int,
+        structure_exclusions: List[StructureExclusion] = None,
```

Comment:
If it can take `None`, then the appropriate type hint is `Optional[List[StructureExclusion]]`.